config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
I'm finalizing on this paragraph. Is this alright? :

Welcome to the Public Lab Talk Pad. 
This page allows for synchronized editing by multiple people as you type. 
It can be used to help you: 
  - collaborate with fellow wiki editors, 
  - take notes with multiple people in real time, and 
  - log the changes, decisions and notes made about a wiki page, or a topic.
[Learn more](publiclab.org/wiki/talk-pages)